packages/playwright-core/src/server/browserContext.ts

```diff
@@ -276,6 +276,22 @@ export abstract class BrowserContext extends SdkObject {
     return await this.doGetCookies(urls as string[]);
   }
 
+  async removeCookies(cookies: {name?: string, domain?: string, path?: string}): Promise<void> {
```

Comment:
somehow I feel like `cookies` as name of the argument is misleading
I would expect array of cookies or something
maybe naming it `criteria` or something similar?

Reply:
I second this, `filter` would probably be a better name, but we can fix it during the pre-release api review.